test/integration/sites_integration_test.rb

```diff
@@ -19,7 +19,7 @@ class SitesIntegrationTest < ActionDispatch::IntegrationTest
     assert_redirected_to 'http://lvh.me/'
     # If there's a match, display a custom homepage
     get 'http://hulme.lvh.me'
-    assert_template 'sites/default.html.erb'
```

Comment:
The other render changes fix deprecation warnings about referring to templates with dots in them (Rails seems to infer the fact that it's an ERB template file). This fixes the test so that it matches the newly changed code.